torch/_inductor/triton_heuristics.py

```diff
@@ -514,7 +509,17 @@ def pointwise(size_hints, meta, tile_hint=None, filename=None):
     bs = max(256, min(numel // 128, 1024))
 
     if len(size_hints) == 1:
-        return cached_autotune([triton_config(size_hints, bs)], meta=meta)
+        configs = [triton_config(size_hints, bs)]
+        if config.max_autotune:
+            configs.extend([
+                # improve 1.832x for https://gist.github.com/shunting314/69b5055193148ade349ac7e58c85d2d9 
```

Comment:
this case is probably spilling a lot with default config, we should check spills when we compile and recompile with more warps, even when max_autotune is not on.

Reply:
yep, I also checked the kernel's triton code. It does a lot of things. The default config simply based on numel can pick a too large block size.

I'll followup with a separate PR to consider number of spills